yb-voyager/src/queryissue/queryissue.go

```diff
@@ -57,6 +58,13 @@ func (p *ParserIssueDetector) GetAllIssues(query string) ([]issue.IssueInstance,
 			}
 			issues = append(issues, issuesInQuery...)
 		}
+
+		percentTypeSyntaxIssues, err := p.GetPercentTypeSyntaxIssues(query)
```

Comment:
is `%TYPE` valid syntax ONLY in functions/procedures, or is it possible to use it in other contexts?

Reply:
As per this [docs](https://www.postgresql.org/docs/current/sql-createfunction.html#:~:text=The%20type%20of%20a%20column%20is%20referenced%20by%20writing%20table_name.column_name%25TYPE.%20Using%20this%20feature%20can%20sometimes%20help%20make%20a%20function%20independent%20of%20changes%20to%20the%20definition%20of%20a%20table), it looks like the feature is for Functions and Procedures. I even tried with other DDLs like TABLE and TYPE, it didn't work - 
```
schema_changes=> create type test_type as (id int, d combined_tbl.d%TYPE);
ERROR:  syntax error at or near "%"
LINE 1: create type test_type as (id int, d combined_tbl.d%TYPE);
                                                      ^
schema_changes=> create type test_type as (id int, d daterange);
CREATE TYPE
schema_changes=> create table tt(id int, d combined_tbl.d%TYPE);
ERROR:  syntax error at or near "%"
LINE 1: create table tt(id int, d combined_tbl.d%TYPE);
                                                ^
schema_changes=> create table tt(id int, d daterange);
CREATE TABLE
schema_changes=> 
```